docs/guides/integration/github.md

```diff
@@ -239,7 +191,40 @@ steps:
 
 It may improve CI times to store uv's cache across workflow runs.
 
-The cache can be saved and restored with the official GitHub `cache` action:
+The [`astral-sh/setup-uv`](https://github.com/astral-sh/setup-uv) has built-in support for
+persisting the cache:
+
+```yaml title="example.yml"
+- name: Enable caching
+  uses: astral-sh/setup-uv@v1
+  with:
+    enable-cache: true
+```
+
+You can configure the action to use a custom cache directory:
```

Comment:
```suggestion
You can configure the action to use a custom cache directory on the runner:
```

What about being more explicit here so people know we don't mean the GitHub Actions Cache?